src/NUnitFramework/framework/Constraints/Comparers/ArraysComparer.cs

```diff
@@ -0,0 +1,62 @@
+﻿// ***********************************************************************
+// Copyright (c) 2009 Charlie Poole, Rob Prouse
```

Comment:
Don't bother to change, but for future reference, we tend to update the copyright to this year for new files or major changes.